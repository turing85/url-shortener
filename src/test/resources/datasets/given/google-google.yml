shortened_url:
- url: "https://google.com"
  shortened: "google"