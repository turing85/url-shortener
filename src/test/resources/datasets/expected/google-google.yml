shortened_url:
- id: "regex:\\d+"
  url: "https://google.com"
  shortened: "google"